Jenkinsfile

```diff
@@ -177,6 +215,38 @@ stage('Build without sync') {
         stash includes: 'wrappers/build/**/*.dll', name: 'uwp-wrappers-nosync'
       }
     },
+    'macOS': {
+      nodeWithCleanup('osx') {
+        getArchive()
+
+        dir('wrappers') {
+          cmake 'build-osx', "${pwd()}/build", configuration
+        }
+
+        stash includes: "wrappers/build/Darwin/${configuration}/**/*", name: 'macos-wrappers-nosync'
```

Comment:
I wonder if it makes sense to ship nosync wrappers for OSX and Linux. I guess we might just as well do it, but I don't expect people to be upset if we didn't :)